src/logic/tokens/api/fetchTokenBalanceList.js

```diff
@@ -9,7 +9,7 @@ const fetchTokenBalanceList = (safeAddress: string) => {
 
   return axios.get(url, {
     params: {
-      limit: 300,
+      limit: 3000,
```

Comment:
does it makes sense to move this to a constant, so if we change there it will be reflected in fetchTokenBalanceList and fetchTokenList?